packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -52,6 +52,7 @@ function LoginForm({
 
   return (
     <AuthForm
+      aria-label="Login using email and password"
```

Comment:
Made this to give a name to the form, this should be mandatory but, as a general optimization,  we can also make it point with `aria-labelledby` to the form title if we are 100% sure to have one